crc/scripts/delete_file.py

```diff
@@ -11,10 +11,13 @@
 class DeleteFile(Script):
 
     @staticmethod
-    def process_document_deletion(doc_code, workflow_id, task):
+    def process_document_deletion(doc_code, workflow_id, task, study_wide=True):
         if DocumentService.is_allowed_document(doc_code):
-            result = session.query(FileModel).filter(
-                FileModel.workflow_id == workflow_id, FileModel.irb_doc_code == doc_code).all()
+            query = session.query(FileModel)\
```

Comment:
whew,  maybe we should add a filter for the study id?!?!  Won't this delete every file with that doc code?